packages/adminrouter/extra/src/common/http.conf

```diff
@@ -35,6 +46,7 @@ init_by_lua_block {
     auth = require "common.auth.open".init(os.getenv("ADMINROUTER_ACTIVATE_AUTH_MODULE"));
     -- Initialize cache:
     cache = require "master.cache".init(nil);
+    historyservice = require "master.history-service".init();
```

Comment:
No, this module/library is later used in `rewrite_by_lua` to establish the destination for `proxy_pass` directive in `location /dcos-history-service/` block.